addons/mass_mailing_crm/models/mailing_mailing.py

```diff
@@ -24,12 +24,20 @@ def _compute_crm_lead_count(self):
             mass_mailing.crm_lead_count = mapped_data.get(mass_mailing.source_id.id, 0)
 
     def action_redirect_to_leads_and_opportunities(self):
-        view = 'crm.crm_lead_all_leads' if self.use_leads else 'crm.crm_lead_opportunities'
-        action = self.env.ref(view).sudo().read()[0]
-        action['view_mode'] = 'tree,kanban,graph,pivot,form,calendar'
-        action['domain'] = [('source_id', 'in', self.source_id.ids)]
-        action['context'] = {'active_test': False, 'create': False}
-        return action
+        text = _("Leads") if self.use_leads else _("Opportunities")
+        helper_header = _(f"No {text} yet!")
+        helper_message = _("Make sure to include a link to a form or a reply-to an " +
+                           f"alias to generate {text} from this mailing.")
+        return {
+            'context': {'active_test': False, 'create': False, 'search_default_day': True},
+            'domain': [('source_id', 'in', self.source_id.ids)],
+            'help': f'<p class="o_view_nocontent_smiling_face">{helper_header}</p><p>{helper_message}</p>',
```

Comment:
fstring doesn't work with translation. please use `_("your %s is your brol", brol)` instead

also, not sure this sentence is super clear : 
`Make sure to include a link to a form or a reply-to an alias to generate {text} from this mailing.`
Maybe reword a little ? @LNAodoo